logic/peer.js

```diff
@@ -42,12 +52,14 @@ Peer.prototype.properties = [
 	'height',
 	'clock',
 	'updated',
-	'nonce'
+	'nonce',
+	'httpPort'
 ];
 
 Peer.prototype.immutable = [
 	'ip',
 	'port',
```

Comment:
it would be actually massive change as it used in many different places. Does anyone else agree?